core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java

```diff
@@ -99,4 +100,21 @@ public void testLoadKlaytnWalletKeyNormal() {
         assertEquals("0xee135d0b57c7ff81b198763cfd7c43f03a5f7622", credentials2.getAddress());
     }
 
+    @Test
+    public void testValidPrivateKey() throws Exception {
+        String longPrivateKey = Keys.createEcKeyPair().getPrivateKey().toString(16) + "00000000000000000000000000000000000000000000000000000000000000000";
+        assertEquals("longPrivateKey: " + longPrivateKey, false, KlayWalletUtils.isValidPrivateKey(longPrivateKey));
```

Comment:
How about this assert expression like below. I think it's more easy to read code.
```suggestion
        assertFalse("longPrivateKey: " + longPrivateKey, KlayWalletUtils.isValidPrivateKey(longPrivateKey));
```